lib/account/account-manager.js

```diff
@@ -5,6 +5,23 @@
  */
 
 /**
+ * Information about an account holder.
+ *
+ * @typedef {Object} accountManager~accountFile
+ * @property {string} email
+ * @property {string} mfaKey
+ * @property {string} password
+ * @property {string} passwordSalt
```

Comment:
Just confirming, this is the Base64 encoded buffer, yes?

Comments in this file about the data stored for the properties, especially for this particular field, would be very helpful.